metadata generator fails when a input parameter contains "password" string

I am working in Terraform on the Postgres ICD module, and there is an input parameter "adminpassword" (https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/database#adminpassword). When I use this input parameter then metadata generation fails with the following error and it wipes out all the "module-metadata.json" file content.

```
Generate terraform module metadata.......................................Failed
- hook id: terraform-config-inspect
- files were modified by this hook

panic: interface conversion: interface {} is bool, not *bool

goroutine 1 [running]:
github.com/IBM-Cloud/terraform-config-inspect/tfconfig.ExtractVariableMetadata(0xc0003d4300, 0x13317e0, 0xc00105e120, 0xc0000b8ba0, 0xc, 0xc0000b8cd0, 0xd)
        github.com/IBM-Cloud/terraform-config-inspect/tfconfig/load.go:594 +0x1366
github.com/IBM-Cloud/terraform-config-inspect/tfconfig.findVariableMetadataFromResourceOrDatasource(0x13796d1, 0x8, 0xc0000f8e10, 0xc0000f8d50, 0xc000098300)
        github.com/IBM-Cloud/terraform-config-inspect/tfconfig/load.go:505 +0x3bf
github.com/IBM-Cloud/terraform-config-inspect/tfconfig.LoadIBMModule(0x137868f, 0x1, 0x205ae878b, 0x63, 0xc00007fd70, 0x0, 0x0, 0xc0000b3cd0, 0x12fcab4)
        github.com/IBM-Cloud/terraform-config-inspect/tfconfig/load.go:98 +0x43f
github.com/IBM-Cloud/terraform-config-inspect/tfconfig.CheckForInitDirectoryAndLoadIBMModule(0x137868f, 0x1, 0x205ae878b, 0x63, 0x0, 0x0, 0x0, 0x60)
        github.com/IBM-Cloud/terraform-config-inspect/tfconfig/load.go:50 +0x43d
main.main()
        github.com/IBM-Cloud/terraform-config-inspect/main.go:33 +0xe5
```

--- a/tfconfig/load.go
+++ b/tfconfig/load.go
@@ -591,7 +591,8 @@ func ExtractVariableMetadata(v *Variable, m interface{}, moduleName, moduleAttri
 					v.Optional = &optional
 				}
 				if a, ok := arg["secure"]; ok && v.Sensitive == nil {
-					v.Sensitive = a.(*bool)
+					sensitive := a.(bool)
+					v.Sensitive = &sensitive
 				}
 				if a, ok := arg["deprecated"]; ok && v.Deprecated == "" {
 					v.Deprecated = a.(string)

